db/queries/application/queries.py

```diff
@@ -44,9 +43,10 @@ def get_application(app_id, include_forms=False, as_json=False) -> dict | Applic
 
 
 def get_applications(
-    filters=[], include_forms=False, as_json=False
+    filters=None, include_forms=False, as_json=False
 ) -> list[dict] | list[Applications]:
-
+    if filters is None:
+        filters = []
```

Comment:
Refactored the function to use None as the default value for mutable arguments (like lists) and initialize them inside the function. This change ensures that each function call gets a fresh instance of the mutable object, preventing unintended shared state across calls. This is a recommended best practice in Python to avoid unexpected behavior with mutable default arguments.